core/domain/question_services.py

```diff
@@ -0,0 +1,79 @@
+# coding: utf-8
+#
+# Copyright 2017 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Services for questions data model."""
+
+from core.platform import models
+
+import feconf
+
+(question_models,) = models.Registry.import_models([models.NAMES.question])
+
+# This takes additional 'title' parameters.
+CMD_CREATE_NEW = 'create_new'
+
+def _create_question(committer_id, question, commit_message, commit_cmds):
+    """Creates a new question.
+
+    Args:
+        committer_id: str. ID of the committer.
+        question: Question. question domain object.
+        commit_message: str. A description of changes made to the question.
+        commit_cmds: list(dict). A list of change commands made to the given
+            question.
+    """
+    model = question_models.QuestionModel.create(
+        title=question.title,
+        question_data=question.question_data,
+        data_schema_version=question.data_schema_version,
+        collection_id=question.collection_id,
+        language_code=question.language_code,
+    )
+
+    model.commit(committer_id, commit_message, commit_cmds)
+    return model
+
+
+def add_question(committer_id, question):
+    """Saves a new question.
+    Args:
+        committer_id: str. ID of the committer.
+        question: Question. QUuestion to be saved.
+    """
+    commit_message = (
+        'New question created with title \'%s\'.' % question.title)
+    question_model = _create_question(committer_id, question, commit_message, [{
+        'cmd': CMD_CREATE_NEW,
+        'title': question.title,
+    }])
+
+    return question_model
+
+def delete_question(committer_id, question_id, force_deletion=False):
```

Comment:
Please leave two lines above top-level functions and classes. Ditto elsewhere throughout this PR.